### PR TITLE
Test preselector

### PR DESCRIPTION
**Pull request by ocefpaf**
Closes #5

### CODE REVIEW

File: recipe/meta.yaml [hunk 1]
```diff
@@ -24,7 +24,7 @@ requirements:
 
 test:
     requires:
-        - python {{ environ['PY_VER'] + '*' }}  # [win]
+        - python {{ PY_VER ~ '*' }}  # [win]
```
Comment (pelson) [inline recipe/meta.yaml]: Out of interest, what is the motivation @ocefpaf?

Comment (ocefpaf) [inline recipe/meta.yaml]: The bug in the latest `conda-build` that prevent us from using  `environ['PY_VER']` as that is recognized as another selector.

Comment (ocefpaf) [inline recipe/meta.yaml]: BTW this is the same fix that @jakirkham did in https://github.com/conda-forge/hdf5-feedstock/pull/13/files, but a different approach. I'd rather if we settle on version only. (I don't have a preferred version though as less typing is good, but that line is quite obscure.)

Comment (ocefpaf) [inline recipe/meta.yaml]: BTW we might not need any change at all:

https://github.com/conda/conda-build/pull/957

Comment (pelson) [inline recipe/meta.yaml]: Another fix. `environ.get('PY_VER')`.

I'd go for the latter until the conda-build fix is available to us.

**Comment by jakirkham** [comment]
Just as FYI, there should be a `conda-build` 1.20.3 release later today. So, maybe it isn't worth getting to hung up on how we handle this issue.

**Pull request merged**